modules/administration/pages/tshoot-taskomatic.adoc

```diff
@@ -0,0 +1,34 @@
+[[troubleshooting-taskomatic]]
+= Troubleshooting Taskomatic
+////
+PUT THIS COMMENT AT THE TOP OF TROUBLESHOOTING SECTIONS
+
+Troubleshooting format:
+
+One sentence each:
+Cause: What created the problem?
+Consequence: What does the user see when this happens?
+Fix: What can the user do to fix this problem?
+Result: What happens after the user has completed the fix?
+
+If more detailed instructions are required, put them in a "Resolving" procedure:
+.Procedure: Resolving Widget Wobbles
+. First step
+. Another step
+. Last step
+////
+
+If Taskomatic crashes or has failed to terminate a process, repository metadata regeneration can be prevented from happening regularly.
+When this occurs, package updates will be available in the {webui}, but will not appear on the client, and attempts to update the client will fail.
+To correct this, determine if any processes are running, or if a crash could have occurred.
+When you have located the problematic process, the metadata can be regenerated and synchronization occurs as expected.
+
+.Procedure: Resolving Taskomatic Problems
+
+. On the server, check the [path]``/var/log/rhn/rhn_taskomatic_daemon.log`` file to determine if any processes are still running or a crash occurred.
```

Comment:
```suggestion
. On the server, check the [path]``/var/log/rhn/rhn_taskomatic_daemon.log`` file to determine if any metadata regeneration processes are still running or a crash occurred.
```

Reply:
I think it would be good to add an example of how such logs look like:

Start of metadata regeneration looks like this:

```
<START DATE> <START TIME> [Thread-<NUMBER>] INFO  com.redhat.rhn.taskomatic.task.repomd.RepositoryWriter - Generating new repository metadata for channel '<CHANNEL>'(sha256) <PACKAGE COUNT> packages, <PATCH COUNT> errata
```

End looks like this:
```
<END DATE> <END TIME> [Thread-<NUMBER>] INFO  com.redhat.rhn.taskomatic.task.repomd.RepositoryWriter - Repository metadata generation for '<CHANNEL>' finished in <DURATION> seconds
```

Example:

```
2019-09-19 12:20:00,174 [Thread-584] INFO  com.redhat.rhn.taskomatic.task.repomd.RepositoryWriter - Generating new repository metadata for channel 'cloned-2018-q1-sles12-sp3-updates-x86_64'(sha256) 550 packages, 140 errata

...

2019-09-19 12:20:04,704 [Thread-584] INFO  com.redhat.rhn.taskomatic.task.repomd.RepositoryWriter - Repository metadata generation for 'cloned-2018-q1-sles12-sp3-updates-x86_64' finished in 4 seconds
```